src/test/java/com/wojciechbarwinski/demo/epic_board_games_shop/services/OrderServiceFacadeTest.java

```diff
@@ -0,0 +1,65 @@
+package com.wojciechbarwinski.demo.epic_board_games_shop.services;
+
+import com.wojciechbarwinski.demo.epic_board_games_shop.dtos.AddressDTO;
+import com.wojciechbarwinski.demo.epic_board_games_shop.dtos.OrderLineDTO;
+import com.wojciechbarwinski.demo.epic_board_games_shop.dtos.OrderRequestDTO;
+import com.wojciechbarwinski.demo.epic_board_games_shop.dtos.OrderResponseDTO;
+import com.wojciechbarwinski.demo.epic_board_games_shop.entities.OrderStatus;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest
+class OrderServiceFacadeTest {
+
+
+    @Autowired
+    private OrderServiceFacade orderServiceFacade;
+
+    @Test
+    @WithMockUser(username = "owner@mail.com")
```

Comment:
Nie mam pewnościjakby się to zachowywało w sytuacji odpalenia x testów naraz (na kilku wątkach)